platform/jobs/edxPlatformBokChoyMaster.groovy

```diff
@@ -0,0 +1,178 @@
+package devops
+
+import hudson.model.Build
+import static org.edx.jenkins.dsl.JenkinsPublicConstants.JENKINS_PUBLIC_LOG_ROTATOR
+import static org.edx.jenkins.dsl.JenkinsPublicConstants.JENKINS_PUBLIC_WORKER
+import static org.edx.jenkins.dsl.JenkinsPublicConstants.JENKINS_PUBLIC_PARSE_SECRET
+import static org.edx.jenkins.dsl.JenkinsPublicConstants.JENKINS_PUBLIC_HIPCHAT
+import static org.edx.jenkins.dsl.JenkinsPublicConstants.JENKINS_PUBLIC_BASE_URL
+
+/*
+Example secret YAML file used by this script
+publicJobConfig:
+    open : true/false
+    jobName : name-of-jenkins-job-to-be
+    url: github-url-segment
+    url-clone : github-url-segment-clone
```

Comment:
nit: perhaps better variable naming here. either numbered urls (url1, url2) or something like 'platformUrl'/'testengUrl', just to avoid confusion. Not super important